CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -9,9 +9,20 @@
 
 See [code changes](https://github.com/aws/aws-k8s-tester/compare/v1.4.5...v1.4.6).
 
+### `eksconfig`
+
+- Add [`SkipDeleteClusterAndNodes` to skip EKS cluster and nodes deletion](https://github.com/aws/aws-k8s-tester/commit/565635179860b4832b64cbe3b39fdbe1c12b1ae1).
+  - Useful for testing add-ons with existing clusters.
+  - If true, delete operation keeps all resources created before cluster (e.g. IAM role, VPC, CMK, etc.).
+  - Set via `AWS_K8S_TESTER_EKS_SKIP_DELETE_CLUSTER_AND_NODES=true`.
+  - Default `AWS_K8S_TESTER_EKS_SKIP_DELETE_CLUSTER_AND_NODES=false`.
+  - `aws-k8s-tester eks create cluster --path` must be passed a valid configuration file with existing cluster information in order to use the existing cluster. Create cluster once, and delete all add-ons, and keep the cluster related fields in the YAML with all other fields removed.
```

Comment:
with this path to yaml eventually be our custom resource yaml config?

Reply:
Yes.